### PR TITLE
adding Client sample for retrieving BOM component vulnerability info

### PR DESCRIPTION
**Pull request by gsnyder2007**

### CODE REVIEW

File: examples/client/get_bom_component_vuln_info.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+with open(args.token_file, 'r') as tf:
+    access_token = tf.readline().strip()
+
+bd = Client(base_url=args.base_url, token=access_token, verify=args.verify)
+
+params = {
+    'q': [f"name:{args.project_name}"]
+}
+projects = [p for p in bd.get_resource('projects', params=params)]
```
Comment (skiyooka) [inline examples/client/get_bom_component_vuln_info.py:29]: Add here: if p['name'] == args.project_name

File: examples/client/get_bom_component_vuln_info.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    pprint(bd.list_resources(vuln_details))
+    if 'related-vulnerability' in bd.list_resources(vuln_details):
+        related_vuln = bd.get_resource("related-vulnerability", vuln_details, items=False)
+    else:
+        related_vuln = None
+    bom_component_vuln['related_vulnerability'] = related_vuln
+    all_bom_component_vulns.append(bom_component_vuln)
+
+pprint(bom_component_vuln)
```
Comment (skiyooka) [inline examples/client/get_bom_component_vuln_info.py:60]: should be all_bom_component_vulns or indented?

**Review by skiyooka — COMMENTED** [review]
I had to add the following because a certain vuln (/api/vulnerabilities/CVE-2014-0114) had too many (i.e. more than 100) headers:
```
import http.client
http.client._MAXHEADERS = 1000
```

**Comment by skiyooka** [comment]
Overall looks nice @gsnyder2007, can you make a couple of quick edits and then go ahead an merge it in.

File: examples/client/get_bom_component_vuln_info.py (hunk 1 above)
Comment (gsnyder2007) [reply examples/client/get_bom_component_vuln_info.py]: thanks, done.

File: examples/client/get_bom_component_vuln_info.py (hunk 2 above)
Comment (gsnyder2007) [reply examples/client/get_bom_component_vuln_info.py]: Yes, fixed.

**Pull request merged**